setup.py

```diff
@@ -50,6 +54,7 @@ def read(filename):
         irclog2html = irclog2html.irclog2html:main
         logs2html = irclog2html.logs2html:main
         irclogsearch = irclog2html.irclogsearch:main
+        serve = irclog2html.irclogsearch:serve
```

Comment:
Actually, I may have misunderstood.  If the purpose of this script is just for manual testing while developing, maybe remove the entry point from setup.py and [define it explicitly in buildout.cfg](https://pypi.python.org/pypi/zc.recipe.egg/2.0.2#specifying-entry-points)?